packages/xstate-wallet/src/chain.ts

```diff
@@ -325,18 +320,18 @@ export class ChainWatcher implements Chain {
       to: NITRO_ADJUDICATOR_ADDRESS
     };
 
-    const response = await this.signer.sendTransaction(transactionRequest);
+    const response = await this.signer.sendTransaction(transactionRequest as any); // TODO: Seems to be an ethers typing issue around the nonce;
```

Comment:
What's the issue exactly?

Reply:
https://github.com/ethers-io/ethers.js/issues/844

Reply:
This may be due to different implementations of `BigNumber` in the repo